cimetrics/plot.py

```diff
@@ -246,3 +399,8 @@ def split(self, series):
     xticks = mtick.FormatStrFormatter(fmt)
     ax.xaxis.set_major_formatter(xticks)
     plt.savefig(os.path.join(metrics_path, "diff.png"))
+
+
+if __name__ == "__main__":
+    env = get_env()
+    (trend_view if env.view == "trend" else default_view)(env)
```

Comment:
Shouldn't this be hidden inside cimetrics?

Reply:
How do you mean?

Reply:
Oh, exposed as an API call as opposed to considering that the CLI is the interface?

Reply:
It just seems weird that the end user has to inspect the env manually whereas all the rest of the env is dealt with by the library.

Reply:
Ok, sure, I'll hide that behind a call. I just didn't expect any library use of plot.